Add documentation for metadata address and how to handle invites to the backoffice (#6996)
umbraco/UmbracoDocs: umbraco-cloud/set-up/external-login-providers.md

> </td><td>A known case for using a special metadata address is when using Entra Id, and you have set up Claims mapping. In this case, you need to set the metadata address to the following:

Reviewer: ```suggestion
</td><td>A common scenario for using a special metadata address is when working with Entra ID and configuring claims mapping. In this case, you must set the metadata address to the following:
```

<code>https://login.microsoftonline.com/{tenant}/v2.0/.well-known/openid-configuration?appid={client-id}</code></td></tr></tbody></table>

### Handling invites when using an External Login Provider

> When using an External Login Provider, we are no longer able to handle the invitation flow to the backoffice. This is because the user needs to be created in the External Login Provider in order to log in. Umbraco Cloud can not handle this integration. 
> So when using an External Login Provider, it is up to you as an administrator to handle how users get access to the backoffice. 

Reviewer: ```suggestion
When using an External Login Provider, the invitation flow to the backoffice can no longer be managed within Umbraco. This is because users must first be created in the External Login Provider before they can log in. Umbraco Cloud does not handle this integration.

As an administrator, you are responsible for managing user access to the backoffice.
```

> A simple way is to send them an email containing a link to the backoffice, telling them to click the "Login with [your login provider]" button. 

Reviewer: ```suggestion
Send users an email with a backoffice link, instructing them to click "Login with [your login provider]".
```
